src/dialogs/confirmation/show-dialog-confirmation.ts

```diff
@@ -0,0 +1,21 @@
+import { fireEvent } from "../../common/dom/fire_event";
+
+export interface ConfirmationDialogParams {
+  title?: string;
+  text: string;
+  confirm: () => void;
```

Comment:
For the future, we should add a dismiss callback, and the ability to overwrite the `OK` and 'Cancel' buttons text.